vocab/credentials/v2/vocabulary.yml

```diff
@@ -17,160 +17,145 @@ ontology:
+    comment: The value of the property must be a URL to the service endpoint associated with the subject. <b><em>There is no URL yet in a VCWG document to <em>define</em> this term.</em></b> 
 
   - id: termsOfUse
-    label: terms of use
-    domain: cred:VerifiableCredential,cred:VerifiablePresentation
-    range: odrl:Policy
-    comment: |
-      If specified, the value of the optional `termsOfUse` property MUST specify one or more terms of use policies under which the issuer issued the credential or presentation. Each `termsOfUse` policy MUST specify its type (for example, `IssuerPolicy`) and MAY specify its instance `id`. The precise content of each term of use is determined by the specific `TermsOfUse` type definition. If the recipient (a holder or verifier) violates the specified terms of use, the responsibility is their own, and such violation may incur legal liability.
+    label: Terms of use
+    defined_by: https://www.w3.org/TR/vc-data-model-2.0/#defn-termsOfUse
+    domain: [cred:VerifiableCredential, cred:VerifiablePresentation]
+    status: reserved
+    range: TermsOfUse
 
   - id: validFrom
     label: Valid from
-    domain: cred:VerifiableCredential
+    domain: [cred:VerifiableCredential, cred:VerifiablePresentation]
+    defined_by: https://www.w3.org/TR/vc-data-model-2.0/#defn-validFrom
     range: xsd:dateTime
-    comment: |
-      The value of the `validFrom` property MUST be a string value of an ISO8601 combined date and time string representing the date and time the credential was issued. Note that this date represents the earliest date when the information associated with the <a href="#credentialSubject>`credentialSubject`</a> property became valid.
 
   - id: validUntil
+    defined_by: https://www.w3.org/TR/vc-data-model-2.0/#defn-validUntil
     label: Valid until
-    domain: cred:VerifiableCredential
+    domain: [cred:VerifiableCredential, cred:VerifiablePresentation]
     range: xsd:dateTime
-    comment: The value of the `validUntil` property MUST be a string value of an ISO8601 combined date and time string representing the date and time the credential ceases to be valid.
 
   - id: verifiableCredential
-    label: verifiable credential
+    defined_by: https://www.w3.org/TR/vc-data-model-2.0/#defn-verifiableCredential
+    label: Verifiable credential
     domain: cred:VerifiablePresentation
     range: cred:VerifiableCredentialGraph
-    comment: The value of the `verifiableCredential` property MUST identify a <a href="#VerifiableCredentialGraph">Verifiable credential graph</a> (informally, it indirectly identifies a <a href="#VerifiableCredential">Verifiable credential</a> contained in a separate graph).
+    comment: The value of the property must identify a <a href="#VerifiableCredentialGraph">Verifiable credential graph</a> (informally, it indirectly identifies a <a href="#VerifiableCredential">Verifiable credential</a> contained in a separate graph).
```

Comment:
I don't understand what this is saying

Reply:
Which part? The main sentence or the parenthetical one?

The value of the property is _not_ a verifiable credential. It is a separate (rdf) graph that contains a (single) verifiable credential. This is what the JSON-LD `"@container": "@graph"` statement in the `@context` file leads to in terms of RDF.